<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZTSubmitDlg</class>
 <widget class="QDialog" name="ZTSubmitDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>475</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>475</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>submit</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="9,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QStackedWidget" name="framesWidget">
         <widget class="QWidget" name="previewPage"/>
         <widget class="QWidget" name="demandPage"/>
         <widget class="QWidget" name="bugPage"/>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>280</number>
       </property>
       <property name="rightMargin">
        <number>280</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNext">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>nextstep</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
